platform/config/Env.ts

```diff
@@ -24,11 +24,8 @@ export class Env {
    * @internal
    */
   constructor(readonly homeDir: string, private readonly argv: WithConfig) {
-    // TODO Fix path, should not be `ts-tmp`, that was only to get stuff running
-    const platformDir = resolve(this.homeDir, 'ts-tmp');
-
```

Comment:
__nit:__ Could you please run prettier, it seems it wants to update few files?

Reply:
✅